.github/workflows/macos.yml

```diff
@@ -0,0 +1,81 @@
+name: continuous-intergration/macos
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+        - v*
+    paths-ignore:
+      - 'README.md'
+      - '.gitlab-ci.yml'
+
+jobs:
+  check:
+    name: build-examples
+    strategy:
+      matrix:
+        platform:
+          - macos-latest
+        toolchain:
+          - nightly
+    runs-on: ${{ matrix.platform }}
+    env:
+      RUST_BACKTRACE: full
+      DELEGATOR_SUBCONTRACTS: "accumulator adder subber"
+    steps:
+
+      - uses: actions/setup-node@v2
+      - run: npm install wasm-opt -g
+
+      - name: Checkout sources & submodules
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+          submodules: recursive
+
+      - name: Install toolchain
+        id: toolchain
+        uses: actions-rs/toolchain@master
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.toolchain }}
+          components: rust-src
+          override: true
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v1.2.0
+
+      - name: Install `cargo-contract` `master`
+        uses: actions-rs/cargo@v1
+        with:
+            command: install
+            args: --git https://github.com/paritytech/cargo-contract.git
+
+      - name: Output versions
+        run: |
+            wasm-opt --version
+            cargo -vV
+            cargo contract --version
+
+      - name: Build examples on ${{ matrix.platform }}-${{ matrix.toolchain }}
+        run: |
+          for example in examples/*/; do
+              echo "Building example: $example";
+              cargo contract build --manifest-path=$example/Cargo.toml;
+          done
+          cd examples/delegator/ && ./build-all.sh
+
+      # For the moment we don't test the examples on MacOS, since it takes a long
```

Comment:
I'd open an issue for this and link to that here instead of leaving commented out code